### PR TITLE
STCOM-330 Update blueprint

### PR DESCRIPTION
**Pull request by cherewaty**
Primary motivation: https://issues.folio.org/browse/STCOM-330

While I was in there, updated `package.json` versions and addressed some linting issues I'd seen in other repos.

### CODE REVIEW

File: resources/ui-app/package.json [hunk 1]
```diff
@@ -12,16 +12,17 @@
...
-    "@folio/stripes-cli": "^1.2.0",
-    "babel-eslint": "^8.0.3",
-    "eslint": "^4.8.0"
+    "@folio/eslint-config-stripes": "^3.2.1",
+    "@folio/stripes-cli": "^1.4.0",
+    "babel-eslint": "^9.0.0",
+    "eslint": "^5.6.0"
   },
   "dependencies": {
     "@folio/stripes-components": "^2.0.0",
```
Comment (ttamj) [inline resources/ui-app/package.json]: Would you like to upgrade to stripes-components to 3.x while you're at it?

File: resources/ui-app/src/routes/example-page.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-import Paneset from '@folio/stripes-components/lib/Paneset';
-import Pane from '@folio/stripes-components/lib/Pane';
-import Button from '@folio/stripes-components/lib/Button';
-import Headline from '@folio/stripes-components/lib/Headline';
+import {
+  Button,
+  Headline,
+  Pane,
+  Paneset
+} from '@folio/stripes-components/lib/Paneset';
```
Comment (ttamj) [inline resources/ui-app/src/routes/example-page.js]: `from '@folio/stripes-components'`

**Comment by ttamj** [comment]
Thanks for the blueprint updates, @cherewaty.  It looks like you have completed STCLI-73 in the process.  @lbigramUWI, FYI.

File: resources/ui-app/package.json (hunk 1 above)
Comment (cherewaty) [reply resources/ui-app/package.json]: Yes I would!

File: resources/ui-app/src/routes/example-page.js (hunk 2 above)
Comment (cherewaty) [reply resources/ui-app/src/routes/example-page.js]: Fixed

**Pull request merged**